python/cuml/test/test_random_forest.py

```diff
@@ -631,13 +612,13 @@ def test_rf_regression_sparse(datatype, split_algo, mode, column_info,
 
 @pytest.mark.memleak
 @pytest.mark.parametrize('fil_sparse_format', [True, False, 'auto'])
-@pytest.mark.parametrize('column_info', [unit_param([100, 50]),
+@pytest.mark.parametrize('column_info', [unit_param([80, 40]),
                          quality_param([200, 100]),
                          stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(1000), quality_param(50000),
+@pytest.mark.parametrize('nrows', [unit_param(800), quality_param(50000),
```

Comment:
While debugging the rf memory issue I had observed that sometimes the memory leakage does not show up when I ran the test for a small number of iterations. What do you think about adding `n_iter` as one of the input parameters and passing a value of 30 or more to it for stress tests?